geometry_msgs/PoseArray poses
int32[] location_ids
int32[] target_ids
int32 timestep
